src/osd/OSD.cc

```diff
@@ -3878,7 +3878,7 @@ int OSD::init()
   osd_op_tp.start();
 
   // start the heartbeat
-  heartbeat_thread.create("osd_srv_heartbt");
+  heartbeat_thread.create("osd_srv_heartbeat");
```

Comment:
Best I can tell this string is arbitrary and not referenced elsewhere, so inclined to accept.